airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/generate_erd/commands.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+from typing import List
+
+import asyncclick as click
+
+from pipelines.airbyte_ci.connectors.consts import CONNECTOR_TEST_STEP_ID
+from pipelines.airbyte_ci.connectors.generate_erd.pipeline import run_connector_generate_erd_pipeline
+from pipelines.cli.dagger_pipeline_command import DaggerPipelineCommand
+from pipelines.helpers.connectors.command import run_connector_pipeline
+from pipelines.helpers.connectors.format import verify_formatters
+
+
+@click.command(
+    cls=DaggerPipelineCommand,
+    short_help="Generate ERD",
+)
+@click.option(
+    "--report",
+    is_flag=True,
+    type=bool,
+    default=False,
+    help="Auto open report browser.",
+)
+@click.option(
+    "--skip-step",
+    "-x",
+    "skip_steps",
+    multiple=True,
+    type=click.Choice([step_id.value for step_id in CONNECTOR_TEST_STEP_ID]),
```

Comment:
For the report, I don't mind. For skipping steps, this is useful. One [example](https://github.com/airbytehq/airbyte/pull/43310/files#diff-62eccd92928fbcd3d285983bfdaa2b0d4ca49016cb9c2f63d6d9fc968c59c541R702)) is documented in the doc. There might be some cases where we don't want to push to dbdocs yet too like in the case you want to test based on the latest change on a branch and see what it would look like without uploading to dbdocs.